lib/modules/apostrophe-images-widgets/public/css/user.less

```diff
@@ -20,5 +20,8 @@
       background-size: contain;
       background-repeat: no-repeat;
     }
+    .apos-empty-image {
```

Comment:
Right now this only affects image widgets in modals. It should apply to those in-context as well.